New questions to GitHub Actions (#238)
FidelusAleksander/ghcertified: content/questions/actions/question-102.md

---

> question: "Which context holds information about the event that triggered a workflow run"

Reviewer: ```suggestion
question: "Which context holds information about the event that triggered a workflow run?"
```

> question: "Select the context property to access information about the event that triggered a workflow run?"

Reviewer: seems more clear

```suggestion
question: "Which context holds information about the event that triggered a workflow run?"
```

Author: updated
